oneoffs/eval_sgf_to_cbt.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Write Minigo eval_game records to Bigtable.
```

Comment:
good place to note why we'd use this.  Current behavior is to already write games, so this is presumably for backfill, yes?  Some usage instructions might be good.